server/src/main/java/org/apache/kylin/rest/DebugTomcat.java

```diff
@@ -99,6 +107,25 @@ private static File findFile(String dir, String ptn) {
         }
         return null;
     }
+    public static boolean isKerberosEnabled() {
+        Configuration conf = new Configuration();
+        boolean authEnabled = conf.getBoolean("hadoop.security.authorization", false);
+        String authMethod = conf.get("hadoop.security.authentication");
+        return authEnabled && authMethod.toLowerCase(Locale.ROOT).equals("kerberos");
+    }
+
+    public static void authKrb5() {
+        System.setProperty("java.security.krb5.conf", "/etc/krb5.conf");
+        Configuration conf = new Configuration();
+        UserGroupInformation.setConfiguration(conf);
```

Comment:
UGI needn't set hadoopconf.org.apache.hadoop.security.UserGroupInformation#isSecurityEnabled will init it.

Reply:
Awesome!